libknet/tests/test-common.c

```diff
@@ -445,18 +445,27 @@ int knet_handle_stop(knet_handle_t knet_h)
 	return 0;
 }
 
-static int _make_local_sockaddr(struct sockaddr_storage *lo, uint16_t offset, int family)
+static int _make_local_sockaddr(struct sockaddr_storage *lo, int offset, int family)
 {
 	uint32_t port;
```

Comment:
Not important, just may make things more obvious/explicit:
```suggestion
	in_port_t port;
```

Reply:
same as above